<!--
Create static transforms to grasp points


Here's one:
((-0.000998465144667643, -0.10012693758088442, 0.24073967922573358),
 (0.8418938925364102,
  -0.4397133161978743,
  0.11772769527839944,
  0.2898397195565091))

Or reversed:
((-0.04040067373878635, -0.21774481173868088, 0.13558550646900713),
 (0.9499580162264987,
  -0.09243589120233672,
  0.01800001736520515,
  -0.2978445446864811))

-->
<launch>
  <node name="ring1_grasp1_tf" pkg="tf" type="static_transform_publisher" output="screen"
    args="-0.04040067373878635 -0.21774481173868088 0.13558550646900713 0.9499580162264987 -0.09243589120233672 0.01800001736520515 -0.2978445446864811 /ring1/ring_link /ring1/grasp1 100"/>

  <node name="ring1_grasp2_tf" pkg="tf" type="static_transform_publisher" output="screen"
    args="-0.147 -0.020 0.191 0.758, -0.647, 0.031, -0.078 /ring1/ring_link /ring1/grasp2 100"/>

  <node name="collab_frame1_creator" pkg="collab_frame_utils" type="rotation.py" output="screen">
    <param name="frame1" value="ring1/ring_link"/>
    <param name="frame2" value="ring1/grasp1"/>
    <param name="verbose" value="0"/>
  </node>

  <node name="collab_frame2_creator" pkg="collab_frame_utils" type="rotation.py" output="screen">
    <param name="frame1" value="ring1/ring_link"/>
    <param name="frame2" value="ring1/grasp2"/>
    <param name="verbose" value="0"/>
  </node>

</launch>
